Source/TextyTextView.swift

```diff
@@ -115,16 +102,14 @@ open class TextyTextView: UITextView, TextStyleDelegate {
         }
         
         if self.style.paragraphStyle == nil {
-            let pstyle: NSMutableParagraphStyle = NSMutableParagraphStyle()
-            pstyle.alignment = .natural
-            pstyle.lineBreakMode = .byWordWrapping
-            self.style.paragraphStyle = pstyle
+            let paragraphStyle: NSMutableParagraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.alignment = .natural
+            paragraphStyle.lineBreakMode = .byWordWrapping
+            self.style.paragraphStyle = paragraphStyle
         }
     }
     
-    internal func didUpdate(style: TextStyle) {
```

Comment:
This was a purely cosmetic change. Using `self.text = self.possiblyTaggedText` seems ambiguous to me since you don't really see what it's doing. Whereas `redrawText()` clearly defines exactly what's happening in that method.

`TextyButton` and `TextyLabel` will be updated as well.